docs/README.md

```diff
@@ -1,3 +1,10 @@
 # Documentation
 
 Each file in this directory (minus this readme) is generated from New Relic NerdGraph GraphQL schema.
+
+## Validation
+
+Every quickstart, datasource, alert, and dashboard is bundled into a single artifact and validated against our [schema](../utils/schema/artifact.json) for correctness.
+
+> [!IMPORTANT]
+> Please ensure that _all_ feature flags are removed from a datasource before it is referenced in a quickstart. Otherwise, the datasource will not be properly associated.
```

Comment:
[nit] Since only core datasources can use FF's, maybe we should consider clarifying this a bit as it could cause confusion for community datasources. I also wonder if we want to be a bit more explicit about the issues caused by not fully enabling a FF.

> For quickstarts referencing a `core` datasource, ensure that _all_ datasource feature flags are removed 100% enabled. Failure to do so will result in the quickstart failing to load properly in the UI.

Not a blocker, just thought maybe it could be a bit more clear.